### ISSUE TITLE
should sp_conv also multipies 1/ratio like the dropout in training mode?

### ISSUE THREAD

**Issue by qianyizhang** [keyu-tian/SparK#12]
this is more of question than issue.

have you tried to implement sp_conv with scale? I believe this may "fix" the disparity in statistics between full conv (which you actually use after pre-train) and sp_conv

https://github.com/keyu-tian/SparK/blob/92aba2d72c12373c84fa8975818c008865964ae8/encoder.py#L22

**Comment by keyu-tian** [comment]
Interesting observation! Yes this has the similarities to Dropout, as both can correct the expectation value of the output (i.e., fix statistics). Furthermore, this operation may not only be effective for our SparK, but also make sense for the field of sparse convolution. We've not attempted it, but I think it's worth trying.
